pypasta/Config.py

```diff
@@ -196,9 +196,9 @@ def path(name):
             mbox_raw = mbox['raw']
             mbox_pub_in = mbox['pubin']
 
-            mbox_pw = 'patchwork'
-            if mbox_pw in mbox:
-                self.pw = Patchwork(mbox[mbox_pw]['url'])
+            mbox_pw = mbox.get('patchwork')
+            if mbox_pw:
+                self.pw = Patchwork(**mbox_pw)
```

Comment:
This patch should be squashed with the patch that introduces the Patchwork class.